lib/components/table/error.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable no-param-reassign */
+import React from 'react';
+import cn from 'classnames';
+import PropTypes from 'prop-types';
+import styles from './error.css';
+import AttentionIcon from '../icon/AttentionIcon';
+
+const VARIANT = {
+  ERROR: 'ERROR',
+  NO_DATA: 'NO_DATA',
+};
+
+const TableError = (props) => {
```

Comment:
I'd like to see a test for this component. Not a big deal though. Feel free to add a story for it.

Reply:
For this repo, I believe we only add stories and then do visual regression testing. And then we test functionality inside the client application.

Reply:
Gotcha. Not sure I'm a fan of that, but if that's the convention, then :+1: